x-pack/legacy/plugins/apm/cypress/tslint.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Also coming from the [examples](https://github.com/cypress-io/cypress-example-recipes/blob/a240054d7f5626ffcd7bd668dded96d219c4a7eb/examples/preprocessors__typescript-webpack/tslint.json). Please take a look 🙏 

BTW, I removed the rule for removing `;`, so please add it back if you consider important to follow project's style.. As I said, this part could be already present in Kibana build, so please discard that part if needed.

Reply:
I think this file can be removed as well?